Tribler/Test/Core/Modules/RestApi/test_upgrader_endpoint.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import
+
+from twisted.internet.defer import inlineCallbacks
+
+from Tribler.Core.Modules.restapi.upgrader_endpoint import SKIP_DB_UPGRADE_STR
+from Tribler.Test.Core.Modules.RestApi.base_api_test import AbstractApiTest
+from Tribler.Test.Core.base_test import MockObject
+from Tribler.Test.tools import trial_timeout
+
+
+class TestShutdownEndpoint(AbstractApiTest):
```

Comment:
Please rename this endpoint and the docstring on the method test_upgrader_skip()

Reply:
Nice (end)point! :wink: